config/rbac/role.yaml

```diff
@@ -241,3 +242,10 @@ rules:
   - securitycontextconstraints
   verbs:
   - use
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - servicemonitors
+  verbs:
+  - create
+  - get
```

Comment:
curious how this was generated? do we need to add `//+kubebuilder:rbac` in metrics.go?